bin/cron/applab_datasets/spotify

```diff
@@ -63,6 +69,13 @@ def get_viral50_data(url)
   return records, columns
 end
 
+def has_profanity?(line)
+  profanity_check = "#{line[2]} - #{line[1]}" # check artist + track name for profanity
+  profanity = ProfanityFilter.find_potential_profanity(profanity_check, PROFANITY_LOCALE)
```

Comment:
Would it be better to do the webpurify check on the whole thing at once, then go through each record and filter as needed based on the results?

Reply:
yeah, i was worried about sending too much data to webpurify, but their limit is 30,000 characters, so we should be safe to do 1 big request. if we go over that we'll know pretty quickly, too 🤷 

i'll refactor to get all of the profanities from the entire dataset, then scan each record to see if it contains that profanity